content/hi/docs/tasks/tools/included/optional-kubectl-configs-bash-linux.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Linux पर bash auto-completion करे"
+description: "Linux पर bash auto-completion के लिए कुछ वैकल्पिक विन्विन्यास।"
+headless: true
+---
+
+ ## परिचय
+
+ बैश के लिए kubectl  कम्प्लिशन स्क्रिप्ट `kubectl completion bash` कमांड के साथ उत्पन्न की जा सकती है।                         आपके शेल में  कम्प्लिशन  स्क्रिप्ट को सोर्स करने से kubectl ऑटोकम्प्लिशन ऐनेबल हो जाती है।
```

Comment:
it is recommended to follow a uniform localising style throughout the doc. 
for example: if we are using `bash` as it is in a certain sentence, its recommended to use the same pattern throughout!